src/main/java/com/twilio/Twilio.java

```diff
@@ -67,6 +69,38 @@ public static void init(final String username, final String password, final Stri
         Twilio.setAccountSid(accountSid);
     }
 
+    /**
+     * Initialize the Twilio environment.
+     *
+     * @param username account to use
+     * @param password auth token for the account
+     * @param region region to make request
+     * @param edge edge to make request
+     */
+    public static void init(final String username, final String password, final String region, final String edge) {
+        Twilio.setUsername(username);
+        Twilio.setPassword(password);
+        Twilio.setRegion(region);
+        Twilio.setEdge(edge);
+    }
+
+    /**
+     * Initialize the Twilio environment.
+     *
+     * @param username account to use
+     * @param password auth token for the account
+     * @param accountSid account sid to use
+     * @param region region to make request
+     * @param edge edge to make request
+     */
+    public static void init(final String username, final String password, final String accountSid, final String region, final String edge) {
```

Comment:
Prefer not to add these 2. Too much overloading. For instance, you can't have just region w/o edge because it would conflict with another constructor (unless you explicitly pass a null edge).